### DISCUSSION TITLE
["ins_rdbms.mk:2: /rdbms/lib/env_rdbms.mk: No such file or directory",

### DISCUSSION THREAD

**Discussion by nsanwar** [ansible-community/community-topics#188] [Closed]
When I try to run make -f ins_rdbms.mk uniaud_on ioracle command it works within server but it is not running via ansible playbook.
I get error ["ins_rdbms.mk:2: /rdbms/lib/env_rdbms.mk: No such file or directory",

**Comment by felixfontein** [comment]
This is the wrong place for such questions. Please check out https://docs.ansible.com/ansible/latest/community/communication.html#asking-questions-over-email where to ask such questions.
